### ISSUE TITLE
Accessibility issue with reference URL

### ISSUE THREAD

**Issue by abhi007tyagi** [saket/Better-Link-Movement-Method#35]
If we pass incomplete reference URLs in <a href> tags and capture the URLs and map them to correct methods or activities, it works. But when accessibility is ON it fails and we get error
`W/URLSpan: Actvity was not found for intent, Intent { act=android.intent.action.VIEW dat=/wps/portal/CONTACT_US (has extras) }`

How to handle such issues?

**Comment by saket** [comment]
This doesn't sound like an issue with this library. Just to confirm: do you run into this error even when using vanilla `LinkMovementMethod`?

**Comment by abhi007tyagi** [comment]
Well, we can't use the vanilla method as that can't help our use case.

**Comment by abhi007tyagi** [comment]
Following method is called in normal use, but in accessibility state ON it is not called
```

method.setOnLinkClickListener((textView, url) -> {
  // Do something with the URL and return true to indicate that 
  // this URL was handled. Otherwise, return false to let the 
  // framework handle the URL.
  return true;
});
```

**Comment by saket** [comment]
Closing this because the issue was unrelated to `BetterLinkMovementMethod`
